cpp/include/copc-lib/las/header.hpp

```diff
@@ -101,9 +102,18 @@ class LasHeader
     double ApplyScaleX(double unscaled_value) const { return unscaled_value * scale_.x + offset_.x; }
     double ApplyScaleY(double unscaled_value) const { return unscaled_value * scale_.y + offset_.y; }
     double ApplyScaleZ(double unscaled_value) const { return unscaled_value * scale_.z + offset_.z; }
-    double ApplyInverseScaleX(double scaled_value) const { return (scaled_value - offset_.x) / scale_.x; }
-    double ApplyInverseScaleY(double scaled_value) const { return (scaled_value - offset_.y) / scale_.y; }
-    double ApplyInverseScaleZ(double scaled_value) const { return (scaled_value - offset_.z) / scale_.z; }
+    int ApplyInverseScaleX(double scaled_value) const
+    {
+        return static_cast<int>((scaled_value - offset_.x) / scale_.x);
+    }
+    int ApplyInverseScaleY(double scaled_value) const
+    {
+        return static_cast<int>((scaled_value - offset_.y) / scale_.y);
+    }
+    int ApplyInverseScaleZ(double scaled_value) const
```

Comment:
good catch, this should probably be a int64_t though.

Reply:
What's the reasoning to force it to int64?